src/main/kotlin/no/nav/syfo/varsel/arbeidstaker/ArbeidstakerVarselApi.kt

```diff
@@ -70,7 +75,7 @@ fun Route.registerArbeidstakerVarselApi(
                     log.warn("$accessDeniedMessage, {}", callIdArgument(callId))
                     call.respond(HttpStatusCode.Forbidden, accessDeniedMessage)
                 }
-            } catch (e: IllegalArgumentException) {
+            } catch (e: Exception) {
```

Comment:
Dette var kanskje en glipp?
Alle Exception er vel ikke en IllegalArgumentException/http400.

Reply:
Koden gjør throw Exception når man ikke finner noe varsel med uuid'en som er sendt inn (både slik den var og slik den er etter min endring), men den burde kanskje gjøre throw IllegalArgumentException i stedet?

Reply:
Ja, om den skal fanges her og be konsument å prøve på nytt med gyldig input.

Reply:
Er det en generisk mekanisme noe sted som gjør om Exception til 500-feil?